File: 백준/Bronze/2985. 세 수/세 수.cs
string[] str = Console.ReadLine().Split();

int a = int.Parse(str[0]);
int b = int.Parse(str[1]);
int c = int.Parse(str[2]);

if (a + b == c)
    Console.WriteLine(a + "+" + b + "=" + c);
else if (a - b == c)
    Console.WriteLine(a + "-" + b + "=" + c);
else if (a * b == c)
    Console.WriteLine(a + "*" + b + "=" + c);
else if (a / b == c)
    Console.WriteLine(a + "/" + b + "=" + c);
else if (a == b + c)
    Console.WriteLine(a + "=" + b + "+" + c);
else if (a == b - c)
    Console.WriteLine(a + "=" + b + "-" + c);
else if (a == b * c)
    Console.WriteLine(a + "=" + b + "*" + c);
else if (a == b / c)
    Console.WriteLine(a + "=" + b + "/" + c);

File: 백준/Bronze/2985. 세 수/세 수.py
a, b, c = map(int, input().split())

if(a+b==c):
    print(a,"+",b,"=",c,sep="")
elif(a-b==c):
    print(a,"-",b,"=",c,sep="")
elif(a*b==c):
    print(a,"*",b,"=",c,sep="")
elif(a/b==c):
    print(a,"/",b,"=",c,sep="")
elif(a==b+c):
    print(a,"=",b,"+",c,sep="")
elif(a==b-c):
    print(a,"=",b,"-",c,sep="")
elif(a==b*c):
    print(a,"=",b,"*",c,sep="")
elif(a==b/c):
    print(a,"=",b,"/",c,sep="")